x-pack/plugins/aiops/public/ui_actions/open_change_point_ml.tsx

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { UiActionsActionDefinition } from '@kbn/ui-actions-plugin/public';
+import { i18n } from '@kbn/i18n';
+import type { CoreStart } from '@kbn/core/public';
+import type { AiopsPluginStartDeps } from '../types';
+import type { EditChangePointChartsPanelContext } from '../embeddable/types';
+import { EMBEDDABLE_CHANGE_POINT_CHART_TYPE } from '../../common/constants';
+
+export const OPEN_CHANGE_POINT_IN_ML_APP_ACTION = 'openChangePointInMlAppAction';
+
+export function createOpenChangePointInMlAppAction(
+  coreStart: CoreStart,
+  pluginStart: AiopsPluginStartDeps
+): UiActionsActionDefinition<EditChangePointChartsPanelContext> {
+  return {
+    id: 'open-change-point-in-ml-app',
+    type: OPEN_CHANGE_POINT_IN_ML_APP_ACTION,
+    getIconType(context): string {
+      return 'link';
+    },
+    getDisplayName: () =>
+      i18n.translate('xpack.aiops.actions.openChangePointInMlAppName', {
+        defaultMessage: 'Open in AIOps labs',
```

Comment:
Nit - I prefer `Open in AIOps Labs` here, to match the way it's displayed in the ML nav.

Reply:
Updated in https://github.com/elastic/kibana/pull/176694/commits/0543699a73e26183efdd84ee6d13269b34e53a5c